yb-voyager/src/dbzm/schemaRegistry.go

```diff
@@ -39,10 +40,14 @@ type TableSchema struct {
 	Columns []Column `json:"columns"`
 }
 
-func (ts *TableSchema) getColumnType(columnName string) (string, error) {
-	for _, colSchema := range ts.Columns {
-		if colSchema.Name == columnName {
-			if colSchema.Schema.Name != "" { // in case Kafka speicfic type which start with io.debezium...
+func (ts *TableSchema) getColumnType(columnName string, targetDBType string) (string, error) {
```

Comment:
Leave it to you whether you want to do it now, or add a TODO for later:
it would be better to add a param that is at the same abstraction level as that of the schemaRegistry. 

so, instead of taking the targetDBType as input, how about you take a boolean flag `returnSourceDbType` which the caller will set to true. 

basically a similar API as that of `debezium.source.datatype.propagate.source.type`

Basically, the main concern being - SchemaRegistry and TableSchema should have no awareness of what the targetDBTYpe is